modules/genomic_browser/php/view_genomic_file.class.inc

```diff
@@ -139,25 +128,5 @@ class View_Genomic_File extends \NDB_Form
             $this->tpl_data['files'][] = $file;
         }
     }
-    /**
-    * Gets the date according to the type
-    *
-    * @param Genomic_File $file  file object
-    * @param string       $type  type of the date
-    * @param array        $array array containing the date
-    *
-    * @return int|string Date (Unix int timestamp) if exists, or an empty string
-    */
-    function _getDate($file, $type, $array)
-    {
-        $pattern  = "/(\d{4})-?(\d{2})-?(\d{2})/";
-        $fileType = $file->getParameter($type);
-        if (preg_match($pattern, $fileType, $array)) {
-            return (mktime(12, 0, 0, $array[2], $array[3], $array[1]));
```

Comment:
Since `$array` depended on the two variables above that do not exist, `mktime()` is effectively being called with (12,0,0,0,0,0) as parameters which generates the output `943981200` (Nov. 30 1999).

Instead, the `AnalysisDate` variable above is retrieved using the getParameter function above like the other variables do.